CHANGELOG.md

```diff
@@ -56,6 +56,7 @@
 - Boot time on AMD achieves the desired performance (i.e under 150ms).
 - Any number of whitespace characters are accepted after ":" when parsing HTTP
   headers.
+- Vsock device restores properly after loading microMV from snapshot.
```

Comment:
Since snapshot isn't part of any previous release, I don't think this CHANGELOG entry makes sense.

Reply:
Entry removed.

Reply:
Since we are now creating releases on separate branches, how does this affect us? Do not you think it is easier to already have all relevant entries in the Changelog and filter them once you do a release?

Reply:
Sure, that should work fine, but for this particular case it doesn't make sense to go in the CHANGELOG under **Fixed** in the first place.

Since the changelog is logs changes between releases, we can't _fix_ something that wasn't broken/didn't exist in the previous release.. this PR is just one of the many PRs from this release that are part of:
```
## Added

- snapshot support
```